src/modals/AuthApp/AuthApp.jsx

```diff
@@ -72,6 +72,7 @@ class AuthApp extends React.Component {
     if (appRequest.isConnected === false) {
       return (
         <div>
+          <Card minHeight={64} title={appRequest.name} image={appRequest.imageURL} color='#a3005c' description={appRequest.description} />
           <p className={styles.description}>
           Cette notification apparait car cette application demande un accès à votre stockage Masq.
```

Comment:
Replace by 'Cette application demande l'accès à votre stockage Masq.' ?
'Cette notification apparait car' may not be useful.

Reply:
ok